infra/events/src/test/java/com/simprints/infra/events/session/SessionEventRepositoryImplTest.kt

```diff
@@ -112,10 +124,13 @@ internal class SessionEventRepositoryImplTest {
     @Test
     fun `return current scope from db if no cache`() = runTest {
         coEvery { eventRepository.getOpenEventScopes(any()) } returns listOf(createSessionScope("mockId"))
+        sessionDataCache.eventCache["test"] = mockk()
+        assertThat(sessionDataCache.eventCache).isNotEmpty()
```

Comment:
This assertion is a bit redundant since you put a value directly into the map on the previous line. It tests only the map itself instead of our business logic.